docker-compose.yml

```
@@ -54,20 +43,15 @@ services:
       - VIRTUAL_HOST
       - LETSENCRYPT_HOST
   mysql:
-   build:
-     context: .docker/mysql
-   volumes:
-     # - ./.docker/mysql/conf:/etc/mysql/conf.d
-     - ./volumes/mysql/dump:/docker-entrypoint-initdb.d
-     - ./volumes/mysql/data:/var/lib/mysql
-   environment:
-     - MYSQL_ROOT_PASSWORD=root
-   ports:
-     - 3306:3306
-   restart: always
-   networks:
-     - mysql
-  #mailhog:
-  #  image: blueimp/mailhog
-  #  ports:
-  #    - 8025:8025
+    build:
+      context: .docker/mysql
+    volumes:
+      - ./volumes/mysql/dump:/docker-entrypoint-initdb.d
+      - ./volumes/mysql/data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=root
```

Every is good to use default values at environments to be possible override by `.env` file
```suggestion
      - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD:-root}
```